src/interface/app.py

```diff
@@ -0,0 +1,31 @@
+import streamlit as st
+
+from typing import List
+from .css import load_css
+from .gui import Gui
+
+
+def create_gui(path_file: str,
+               path_modified_file: str,
+               message_container_height: int,
+               layout_division: List[int]) -> None:
+
```

Comment:
Why did you remove this line?
`st.set_page_config(layout="wide")`
It will help us to keep the whole windows occupied for the GUI.